style.css

```diff
@@ -64,19 +64,24 @@ nav {
 }
 
 .background_image {
-  background-image: url('Images/background_img.png');
+  background-image: url('Images/header-shapes\ mobile.png');
   background-repeat: no-repeat;
   background-size: cover;
   max-width: 100%;
-  height: auto;
+  margin-bottom: 50px;
+  background-position: center;
+  border: 5px;
+  border-radius: 1px 1px 1px 80px;
+  background-color: #fff;
```

Comment:
- [ ] Although I appreciate 🎉🎉👌your effort in adding the background image as suggested by the [previous reviewer](https://github.com/ngalamac/mobile-portfolio/pull/2#discussion_r1214108449), there are still some improvements needed for mobile responsiveness within the range of 375px to 768px. 
- [ ] Currently, your design does not align with the suggested [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=XeKkHNg24zP2w1kF-0) design. 
- [ ] To ensure a consistent and visually appealing experience across different devices, it is important to make the necessary adjustments to your design.🤔🤔
- [ ] Your Design:
![Screenshot 2023-06-03 033611](https://github.com/ngalamac/mobile-portfolio/assets/118761303/df4e11dc-c18c-424a-b5b6-effd151b0530)

- [ ] Figma Design:
![figma](https://github.com/ngalamac/mobile-portfolio/assets/118761303/30ac202c-97ab-4ab0-859b-f9a788b0da16)

- [ ] Kindly align the text as well strictly to the Figma design

Reply:
- [ ] Please do as previously requested. Currently, your design is looking good on the 375px width. 📱 
But we want your webpage to be responsive, won't we? 🤗 Meaning it should look good on any device ☎️ 😁 
Below is an example of how you can achieve that 👇🏽 

```sh 

.cardd {
    width: 100%;
    height: auto;
    background: #fff;
    border-radius: 16px;
    border: 1px solid #def1e6;
    padding: 8px;
    display: flex;
    justify-content: center;
    flex-direction: column;
}

.cardd img {
    width: 100%;
    height: auto;
    border-radius: 8px;
}

```